ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -464,6 +464,12 @@ def __array__(self, dtype=None, /):
         if not dtype:
             return self
         return np_frontend.array(self, dtype=dtype)
+    
+    def __array_wrap__(self, out_arr, context=None):
+        if context is None:
+            return ndarray.view(self)
```

Comment:
For now, I think we should be calling the current instance to return the value `self.view()`

Reply:
Got it.